docs/app/Components/Sidebar/Sidebar.js

```diff
@@ -56,6 +55,7 @@ export default class Sidebar extends Component {
             onChange={this.handleSearchChange}
           />
         </Menu.Item>
+        {addons}
```

Comment:
We were not showing our addons on the doc site.  This includes:

- Confirm
- Textarea
- Select (pending #206)